zulipterminal/ui_tools/buttons.py

```diff
@@ -197,14 +205,15 @@ def __init__(self, user: Dict[str, Any], controller: Any,
 
         # FIXME Is this still needed?
         self.recipients = frozenset({self.user_id, view.model.user_id})
-
+        postfix_caption = (color, '(You)') if is_current_user else ''
```

Comment:
You may have seen in the other work on this issue, that another style was added to the theme and used instead of `color` - I'd suggest we do that here, now that you have the layout working.